src/locales/fi.json

```diff
@@ -92,6 +92,22 @@
       "phone": "Matkapuhelin",
       "email": "Sähköpostiosoite",
       "remarks": "Huomiot"
+    },
+    "customerBill": {
+      "title": "AVOIMET LASKUT",
```

Comment:
This a minor note, but using all capital letters should be moved from translations to styling.